.github/ISSUE_TEMPLATE/new-release.md

```diff
@@ -36,22 +36,22 @@ For Mac:
 
 For Windows:
 
-- [ ] Cleanup repo, build and generate compodocs and publish on github pages (`./scripts/pre-release.sh` - without sampleaddon zip)
-- [ ] Download and rename in root directory `https://github.tools.sap/cx-commerce/spartacussampledataaddon/archive/develop.zip` -> `spartacussampleaddon.zip`
-- [ ] Download and rename in root directory `https://github.tools.sap/cx-commerce/spartacussampledataaddon/archive/develop.tar.gz` -> `spartacussampleaddon.tar.gz`
+- [ ] Cleanup repo, build and generate compodocs and publish on github pages (`yarn generate:docs` and `yarn publish:docs` for patch stable/releases)
+- [ ] Download and rename in root directory `https://github.tools.sap/cx-commerce/spartacussampledataaddon/archive/develop.zip` -> `spartacussampledataaddon.zip`
+- [ ] Download and rename in root directory `https://github.tools.sap/cx-commerce/spartacussampledataaddon/archive/develop.tar.gz` -> `spartacussampledataaddon.tar.gz`
 
 ---
 
-- [ ] Release libraries with release-it scripts (set GITHUB_TOKEN env variable, do `npm login`; for each package select rc1 version when prompted - when prompted):
+- [ ] Release libraries with release-it scripts (set GITHUB_TOKEN env variable, check if you are loggin in with `npm whoami` (if not logged, then login with `npm login`); for each package select version when prompted):
```

Comment:
if version is not suggested it's needed to type it; so let's change the sentence to: "for each package select/type version when prompted"